apps/codelab/src/app/components/breadcrumb/breadcrumb.component.html

```diff
@@ -0,0 +1,8 @@
+<h1>
+  <a routerLink="/">Angular Codelab</a>
```

Comment:
Imho, it's better to add underline style to navigation links

Reply:
I had it initially, but removed because it was too noisy, maybe we should replace the text with an icon eventually?

Reply:
Did try it? If yes, attach a screen, pls)